packages/contracts/contracts/libraries/LibDropStorage.sol

```diff
@@ -10,9 +10,7 @@ library LibDropStorage {
   struct Drop {
     uint256 timeStart;
     uint256 timeEnd;
-    uint256 priceStart;
-    uint256 priceEnd;
-    uint256 stepDuration;
+    uint256 price;
```

Comment:
Because of the immutability of the storage, this is not allowed. We have to keep the existing fields. You only need to add the price at the end of the structure. (you can put a comment on the lines you planned to remove to say it's not used anymore

Reply:
Only allowed to insert at the end, not in the middle it will corrupt the slot